Src/BootCamp.Chapter/CaesarCipher.cs

```diff
@@ -11,12 +11,34 @@ public static class CaesarCipher
     {
         public static string Encrypt(string message, byte shift)
         {
-            return message;
+            var output = ""; 
+
+            if (String.IsNullOrEmpty(message))
+            {
+                return message; 
+            }
+
+
+            foreach (char ch in message)
+            {
+                output += DecryptOneChar(ch, shift); 
+            }
+
+            return output; 
+        }
+
+        private static char DecryptOneChar(char ch, byte shift)
+        {
+            if (!char.IsLetter(ch))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5012532)

Reply:
solved